docker/compose-spec-l2-services.yml

```diff
@@ -5,7 +5,7 @@ services:
   sequencer:
     hostname: sequencer
     container_name: sequencer
-    image: consensys/linea-besu-package:${BESU_PACKAGE_TAG:-mainnet-f2ff06c}
+    image: consensys/linea-besu-package:${BESU_PACKAGE_TAG:-devnet-395ec6d}
```

Comment:
If we're making devnet version the default, we should override them for the V1 tracing to be mainnet

Reply:
Oh, right. You started it in another PR, that's why I missed it